logging/bin/deploy_logging_open_openshift.sh

```diff
@@ -0,0 +1,140 @@
+#! /bin/bash
```

Comment:
Lots of duplication. Any way we can avoid the large number of new scripts that are slight variants of the core files?

Reply:
See note above re:keeping OpenShift-specific in separate scripts for now (as we did on the monitoring side).  For this _particular_ script, given the differences in deployment steps b/w OpenShift and other cloud providers, I'm not sure consolidating into a single script makes sense.